src/lib/operators/jit_operator/jit_types.cpp

```diff
@@ -0,0 +1,65 @@
+#include "jit_types.hpp"
+
+namespace opossum {
+
+template <>
+JitDataType jit_data_type<uint8_t>() {
+  return JitDataType::Bool;
+}
+
+template <>
+JitDataType jit_data_type<int32_t>() {
+  return JitDataType::Int;
+}
+
+template <>
+JitDataType jit_data_type<int64_t>() {
+  return JitDataType::Long;
+}
+
+template <>
+JitDataType jit_data_type<float>() {
+  return JitDataType::Float;
+}
+
+template <>
+JitDataType jit_data_type<double>() {
+  return JitDataType::Double;
+}
+
+template <>
+JitDataType jit_data_type<std::string>() {
+  return JitDataType::String;
+}
+
+template <>
+uint8_t& JitVariantVector::as(const size_t index) {
+  return _bool[index];
```

Comment:
Until now, I thought that `as<bool>` would give me the stored value casted as a bool. This looks to me as if calling `as<bool>` on an int value would be undefined.